core/templating/templating_test.go

```diff
@@ -194,6 +198,80 @@ func Test_ApplyTemplate_CsvCountRowsMissingDataset(t *testing.T) {
 	Expect(template).To(Equal(``))
 }
 
+func Test_ApplyTemplate_CsvSQL_Select(t *testing.T) {
+	RegisterTestingT(t)
+
+	template, err := ApplyTemplate(&models.RequestDetails{}, make(map[string]string), `{{#each (csvSQL "SELECT name FROM test-csv WHERE id == '1'")}}{{this.name}}{{/each}}`)
```

Comment:
`{{#each (csvSQL "SELECT name FROM test-csv WHERE id == '1'")}}` does it work in a simulation response field in which it would be wrapped by another double quotes?

Reply:
I will test. Currently only tested as part of a bodyFile so unescaped

Reply:
Yes, tested and works